docs_user/adoption-attributes.adoc

```diff
@@ -74,6 +76,8 @@ ifeval::["{build}" == "downstream"]
 :OpenShift: Red Hat OpenShift Container Platform
 :OpenShiftShort: RHOCP
 :OpenStackPreviousInstaller: director
+:Ceph: Red Hat Ceph Storage 
+:CephCluster: Red Hat Ceph Storage 
```

Comment:
are Ceph and CephCluster referencing the same content? (wondering if we can use the same variable then)